examples/seed.yaml

```diff
@@ -47,7 +47,7 @@ tags:
 ---
 kind: Function
 name: hello-py
-code: '@examples/python3/hello.py'
```

Comment:
If you change this back, the docs must also reflect this change.  We've caused a lot of headaches because of this change.

Reply:
Looks like the quickstart at least was never updated, so this change would be good.  But make sure we're not missing other places

Reply:
Ok, I'll make sure to check.